Adapter/Translator.php

```diff
@@ -60,7 +60,7 @@ public function __construct(LegacyContext $context)
      *
```

Comment:
+:
You cannot use variables for $id and $domain.
eg: $message = 'hello'; $domain = 'AdminTab'; trans($message,[],$domain); IS FALSE
prefer: trans('hello', [], 'AdminTab'); TO ENSURE TRANS PARSER WILL FIND YOUR MESSAGE.